apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -0,0 +1,253 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import {
+  SfIconShoppingCart,
+  SfIconFavorite,
+  SfIconPerson,
+  SfIconMenu,
+  SfIconChevronRight,
+  SfIconClose,
+  SfButton,
+  SfDrawer,
+  SfListItem,
+  useDisclosure,
+} from '@storefront-ui/react';
+import { useRef } from 'react';
+import { CSSTransition } from 'react-transition-group';
+import brandLogo from '@assets/vsf_logo_white.svg';
+import brandLogoSign from '@assets/vsf_logo_sign_white.svg';
+import watch from '@assets/watch.png';
+
+const actionItems = [
+  {
+    icon: <SfIconShoppingCart />,
+    label: '',
+    ariaLabel: 'Cart',
+    role: 'button',
+  },
+  {
+    icon: <SfIconFavorite />,
+    label: '',
+    ariaLabel: 'Wishlist',
+    role: 'button',
+  },
+  {
+    icon: <SfIconPerson />,
+    label: 'Log in',
+    ariaLabel: 'Log in',
+    role: 'login',
+  },
+];
+
+const bannerDetails = {
+  image: watch,
+  title: 'New in designer watches.',
+};
+
+const categoriesContent = [
+  {
+    heading: 'Activities',
+    items: [
+      {
+        title: 'Fitness',
+        link: '/',
+      },
+      {
+        title: 'Pilates',
+        link: '/',
+      },
+      {
+        title: 'Training',
+        link: '/',
+      },
+      {
+        title: 'Cardio workout',
+        link: '/',
+      },
+      {
+        title: 'Yoga',
+        link: '/',
+      },
+      {
+        title: 'All activities',
+        link: '/',
+      },
+    ],
+  },
+  {
+    heading: 'Categories',
+    items: [
+      {
+        title: 'Clothing',
+        link: '/',
+      },
+      {
+        title: 'Shoes',
+        link: '/',
+      },
+      {
+        title: 'Accessories',
+        link: '/',
+      },
+      {
+        title: 'Wearables',
+        link: '/',
+      },
+      {
+        title: 'Food & Drinks',
+        link: '/',
+      },
+      {
+        title: 'All categories',
+        link: '/',
+      },
+    ],
+  },
+  {
+    heading: 'Womens Deals',
+    items: [
+      {
+        title: 'Fitness Gear',
+        link: '/',
+      },
+      {
+        title: 'Outlet',
+        link: '/',
+      },
+      {
+        title: 'All Womens Deals',
+        link: '/',
+      },
+    ],
+  },
+];
+
+export default function BaseMegaMenu() {
+  const { close, toggle, isOpen } = useDisclosure({ initialValue: false });
+  const drawerRef = useRef(null);
+
+  return (
+    <div className="w-full h-full">
+      {isOpen && <div className="fixed inset-0 bg-neutral-500 bg-opacity-50 transition-opacity" />}
+      <header className="flex justify-center w-full border-0 bg-primary-700 border-neutral-200 h-14 md:relative md:h-20 md:z-10">
+        <div className="flex items-center flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
+          <a href="/" aria-label="SF Homepage" className="inline-block text-white mr-2 lg:mr-10">
+            <picture>
+              <source srcSet={brandLogo.src} media="(min-width: 748px)" />
```

Comment:
This breakpoint value isn't right. There is a gap between 748px and 767px that looks like on a given screenshot. Make sure to adjust it in the NavbarTop component as well.
<img width="403" alt="Screenshot 2023-04-13 at 09 59 30" src="https://user-images.githubusercontent.com/34419118/231693583-fd19afdd-05d0-4e13-8ebb-5d103e3e6742.png">